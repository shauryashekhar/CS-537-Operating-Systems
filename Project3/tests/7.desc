Batch File ends without exit
