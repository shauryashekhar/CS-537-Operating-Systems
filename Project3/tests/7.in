ls -a
echo "This is the last command"
