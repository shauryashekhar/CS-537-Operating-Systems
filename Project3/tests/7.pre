echo "ls -a" > tests/7.out 2> tests/7.err
ls -a >> tests/7.out 2>> tests/7.err
echo "echo \"This is the last command\"" >> tests/7.out 2>> tests/7.err
echo "\"This is the last command\"" >> tests/7.out 2>> tests/7.err
