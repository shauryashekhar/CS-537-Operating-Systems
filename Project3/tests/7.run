./mysh tests/7.in
