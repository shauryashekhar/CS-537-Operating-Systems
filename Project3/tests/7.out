ls -a
.
..
.__afsE5F3
commands
cpplint.out
Makefile
mysh
mysh.c
output
output.c
test
test.c
tester
tests
tests-out
echo "This is the last command"
"This is the last command"
